gnomad_qc/v5/data_ingestion/field_md_to_html.py

```diff
@@ -0,0 +1,115 @@
+import markdown
+from bs4 import BeautifulSoup
+
+# Read markdown content.
+with open("field_requirements.md", "r") as f:
+    markdown_text = f.read()
+
+# Convert markdown to HTML.
+html = markdown.markdown(markdown_text, extensions=["extra"])
+soup = BeautifulSoup(html, "html.parser")
+
+# Color map based on Field Necessity.
+color_map = {
+    "required": "#fff0f0",
+    "optional": "#f0faff",
+    "not_needed": "#dedbe4",
+}
+
+final_sections = []
+first_table_processed = False
+second_table_processed = False
+
+elements = soup.find_all(["p", "table"])
+
+paragraphs_before_first_table = []
+paragraphs_before_second_table = []
+
+for element in elements:
+    if element.name == "p":
+        if not first_table_processed:
+            paragraphs_before_first_table.append(str(element))
+        elif not second_table_processed:
+            paragraphs_before_second_table.append(str(element))
+
+    elif element.name == "table":
+        if not first_table_processed:
+            first_table_processed = True
+            final_sections.append("<h1>Global Fields Specification</h1>")
```

Comment:
this is a naive question: is there a way to parse the HTML for formatting without needing to re-add these headers? it looks like the current code converts the markdown to HTML, so these header lines are correctly converted to h1 headings
![image](https://github.com/user-attachments/assets/e1f1cc38-5c69-4f62-9477-4b8bd6b2463e)